File: app/src/main/java/com/wd/daquan/http/HttpBaseRequestBean.java
package com.wd.daquan.http;

import android.os.Build;

import com.wd.daquan.model.mgr.ModuleMgr;
import com.wd.daquan.model.utils.ModelUtil;

/**
 * HTTP请求基础类
 */
public class HttpBaseRequestBean{
    private String userId = "";
    private String token = "";
    private AppInfo appInfo = new AppInfo();
    private Object data = new Object();
    public HttpBaseRequestBean() {
        setToken(ModuleMgr.getCenterMgr().getToken());
        setUserId(ModuleMgr.getCenterMgr().getUID());
        setAppVersion(ModelUtil.getVersion());
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public void setAppVersion(String version){
        appInfo.setApp_version(version);
    }

    public AppInfo getAppInfo() {
        return appInfo;
    }

    public void setAppInfo(AppInfo appInfo) {
        this.appInfo = appInfo;
    }

    public Object getData() {
        return data;
    }

    public void setData(Object data) {
        this.data = data;
    }

    public class AppInfo{
        private String app_version = "";//app版本
        private String device_type = "android";//设备型号
        private String device_brand = Build.BRAND + Build.MODEL;//设备名称
        private String os_version = Build.VERSION.RELEASE;//系统版本

        public String getApp_version() {
            return app_version;
        }

        public void setApp_version(String app_version) {
            this.app_version = app_version;
        }

        public String getDevice_type() {
            return device_type;
        }

        public void setDevice_type(String device_type) {
            this.device_type = device_type;
        }

        public String getDevice_brand() {
            return device_brand;
        }

        public void setDevice_brand(String device_brand) {
            this.device_brand = device_brand;
        }

        public String getOs_version() {
            return os_version;
        }

        public void setOs_version(String os_version) {
            this.os_version = os_version;
        }
    }

}

File: dqAdSdk/src/main/java/com/dq/sdk/ad/http/bean/HttpBaseRequestBean.kt
package com.dq.sdk.ad.http.bean

import android.os.Build

/**
 * HTTP请求基础类
 */
class HttpBaseRequestBean {
    var appId = "1"
    var sha = "82:69:20:A3:F6:4C:80:DA:5A:3F:BC:C5:67:85:81:F8:28:66:E8:9F"
    var appInfo = AppInfo()
    var data = Any()

    fun setAppVersion(version: String?) {
        appInfo.app_version = version
    }

    inner class AppInfo {
        var app_version: String? = "" //app版本
        var device_type = "android" //设备型号
        var device_brand = Build.BRAND + Build.MODEL //设备名称
        var os_version = Build.VERSION.RELEASE //系统版本

    }

    init {
         appId = "1"
         sha = "82:69:20:A3:F6:4C:80:DA:5A:3F:BC:C5:67:85:81:F8:28:66:E8:9F"
    }
}